main/webapp/modules/core/styles/index/default-importing-sources.css

```diff
@@ -40,3 +40,26 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
   max-width: 50em;
   height: 30em;
 }
+
+.file-input {
+    width: 200px;
+    height: 100px;
+    border: 2px solid #ccc;
+    border-radius: 5px;
+    padding: 5px;
+    box-sizing: border-box;
+}
+
+.file-input-container {
+    position: relative;
+}
+
+.file-input-text {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-110%, -30%);
```

Comment:
How did you come up with such percentages? Will the text stay centered if the width of the box increases? I wonder if there would be a more reliable solution.

Reply:
I came up with these values using trial and error.Will try to find a more reliable solution.

Reply:
The commit 00f4dff makes the text centered along both x and y axis even when the box size increases.